bentoml/_internal/bento/build_config.py

```diff
@@ -308,7 +308,8 @@ def validate_bentofile(yaml_content: t.Dict[str, t.Any]) -> None:
                     logger.warning(
                         "In order to support [bold magenta]scikit-learn[/] on alpine-based images, "
                         "make sure to include the following packages under "
-                        f"`docker.system_packages`: [bold yellow]{', '.join(required_packages)}[/]"
+                        f"`docker.system_packages`: [bold yellow]{', '.join(required_packages)}[/]",
```

Comment:
Could we just check this and error if not found?

Reply:
done